hist/hist/src/TF1Convolution.cxx

```diff
@@ -422,6 +427,10 @@ void TF1Convolution::SetParameters(Double_t p0, Double_t p1, Double_t p2, Double
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+/// Set the fraction of extra range used when doing an FFT convolution
```

Comment:
Stopping the short description (first sentence) of the documentation with a period is particularly important if there are more sentences later, so it's clear where the short description ends for formatting the documentation. That's what I learned from Stephan when he reviewed my PR.

Reply:
Thank you for the suggestion. It is updated now